README.md

```diff
@@ -16,10 +16,40 @@ npm install
 npm start
 ```
 
-- Run debug start:
+- Run debug start (Attach mode):
 
 ```bash
 npm run start:debug
 ```
 
-- And run .vscode/launch.json
+- And run .vscode/launch.json (Debug Attach)
+
+# Debugging state
+
+Since VS Code v1.47, it enables a new debugging feature, [New JavaScript Debugger](https://code.visualstudio.com/updates/v1_47#_debugging) as default.
+
+This new debugger works with no configuration on this way:
```

Comment:
This new debugger works with zero config:

Reply:
Nice